frontend/src/pages/projects/screens/detail/useCheckLogoutParams.ts

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { useSearchParams } from 'react-router-dom';
+import useNotification from '../../../../utilities/useNotification';
+import { ProjectDetailsContext } from '../../ProjectDetailsContext';
+import { getNotebookDisplayName } from '../../utils';
+
+const useCheckLogoutParams = (): void => {
+  const [queryParams, setQueryParams] = useSearchParams();
+  const notification = useNotification();
+  const {
+    notebooks: { data: notebooks },
+  } = React.useContext(ProjectDetailsContext);
+
+  React.useEffect(() => {
+    const notebookLogout = queryParams.get('notebookLogout');
+    if (notebookLogout) {
+      const notebook = notebooks.find((n) => n.notebook.metadata.name === notebookLogout);
+      if (notebook?.notebook) {
+        notification.success(
+          `Logout workbench "${getNotebookDisplayName(notebook.notebook)}" successfully`,
+        );
+        queryParams.delete('notebookLogout');
+        setQueryParams(queryParams);
+      }
```

Comment:
What if it's loaded and you can't find it? Perhaps we put an error case here too? 

Something like:
```
} else if (loaded) {
  notification.error('Error logging out', 'Unable to locate notebook to finalize logout');
  // remove query param
}
```